### ISSUE TITLE
A fileContent resource for simple file contents

### ISSUE THREAD

**Issue by jnury** [PlagueHO/FileContentDsc#5]
Hi,

The 'file' resource of PSDesiredStateConfiguration lake the ability to create file in different encodings (ascii, utf8 without bom etc...). I haven't find any DSC resource that propose to create files in ASCII, so I planned to write a new one to just create a file with the specified content and selected encoding.

As the fileContent resource is already here and propose similar functionalities, would-you mind if I create a PR with a new 'fileContent' resource for simple files ?

**Comment by PlagueHO** [comment]
Hi @jnury - sure! I would gratefully accept the PR! I have adopted the standard contribution policies and style guidelines of the DSC Resource Kit.

**Comment by jcwalker** [comment]
I want to bring this back from the dead @jnury @PlagueHO . 
What do you think about adding an Encoding parameter to all the resources? It's should be a quick win being that Set-Content has an Encoding parameter.

**Comment by PlagueHO** [comment]
HI @jcwalker - I'd be grateful for the assist - I've not had much time of late to get DSC stuff done.

Regarding the expected behavior: If the Encoding parameter was specified, but it differed from the Encoding on the existing file, would you expect that the file encoding was corrected on the file?

I think as a consumer of the resource that a file would be converted to the correct format.

**Comment by jcwalker** [comment]
I agree.  I would think if the desired encoding was different  than the original file it would be converted.

**Comment by jcwalker** [comment]
Are we agreeing that we should move forward with using the Encoding parameter instead of creating a new resource?  I would be happy to own this and could start early next week.

**Comment by PlagueHO** [comment]
@jcwalker - absolutely agree! And I'd love the help if you're up for submitting a PR. I haven't had as much time to put into these projects lately (hands full on the DSC Resource kit).

**Comment by jcwalker** [comment]
@PlagueHO cool!, I plan to submit a PR for this issue and [issue #13 ](https://github.com/PlagueHO/FileContentDsc/issues/13) within the next two weeks

**Comment by nehrua** [comment]
@PlagueHo I discussed this issue with @jcwalker and I'll be picking this up for him.

**Comment by PlagueHO** [comment]
Awesome - thanks @nehrua
